src/main/java/org/broadinstitute/hellbender/utils/plotter/ACNVPlotter.java

```diff
@@ -6,26 +6,29 @@
 /**
  * Calls an R script to create plots of target data
  */
-public final class AlleleFractionSegmentedPlotter {
-    private static final String R_SCRIPT = "AlleleFractionResultsPlotting.R";
+public final class ACNVPlotter {
+    private static final String R_PLOTTING_SCRIPT = "plotting.R";
```

Comment:
consider renaming these?  R_SCRIPT is not extremely descriptive, and neither is plotting.R